examples/stm32g0/Embed.toml

```diff
@@ -0,0 +1,43 @@
+[default.probe]
```

Comment:
examples don't use cargo-embed, please don't add an Embed.toml

Reply:
Will remove. Also change the implementation, not to use the global static signal, but a signal in interrupt state context

Reply:
Added a reset function to pwm to be able to synchronize multiple timers to have the same phase.